run_one_experiment.py

```diff
@@ -244,7 +245,10 @@ def _fuzzing_pipeline(benchmark: Benchmark, model: models.LLM,
     p = pipeline.Pipeline(
         args=args,
         trial=trial,
-        writing_stage_agents=[Prototyper(trial=trial, llm=model, args=args)])
+        writing_stage_agents=[Prototyper(trial=trial, llm=model, args=args)],
+        analysis_stage_agents=[
+            CrashAnalyzer(trial=trial, llm=model, args=args)
+        ])
```

Comment:
Could you please check if the experiment report/bucket is the same as expected?
https://github.com/google/oss-fuzz-gen/pull/814/checks?check_run_id=38069838583